pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -30,16 +30,16 @@ import (
 	utilpointer "k8s.io/utils/pointer"
 )
 
+// TODO: Move these constants to k8s.io/kubelet/config/v1beta1 instead?
```

Comment:
If you end up including this TODO, there should be a bit more context ie the "why" so people can quickly understand. Though this might be better tracked in it's own issue as well.

Reply:
Those TODOs are code left by the original developer. Maybe he/she thinks that is a better format.

Reply:
Ah this diff confused me